Integrations/TruSTAR/TruSTAR.py

```diff
@@ -0,0 +1,752 @@
+        current_report['distributionType'] = current_report_dict['distributionType']
+        if current_report_dict.get('externalUrl'):
+            current_report['externalUrl'] = current_report_dict['externalUrl']
+        current_report['reportBody'] = current_report_dict['reportBody']
+        reports.append(current_report)
+        reports_context.append({
+            'reportTitle': current_report['title'],
+            'reportBody': current_report['reportBody'],
+            'id': current_report['id']
+        })
+    if reports:
+        ec = {
+            'TruSTAR.Report(val.id && val.id === obj.id)': reports_context
+        }
+        title = 'TruSTAR reports'
+        entry = {
+            'Type': entryTypes['note'],
+            'Contents': reports,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown(title, reports),
+            'EntryContext': ec
+        }
+        return entry
+    return 'No reports were found.'
+
+
+def get_correlated_reports(indicators, enclave_ids, distribution_type, page_size, page_number):
+    response = ts.get_correlated_reports_page(indicators, enclave_ids, page_number, page_size)
+    correlated_reports = []  # type: List
+    for report in response:
+        current_report = report.to_dict(remove_nones=True)
+        current_report['updated'] = normalize_time(current_report['updated'])
+        current_report['created'] = normalize_time(current_report['created'])
+        current_report['timeBegan'] = normalize_time(current_report['timeBegan'])
+        return current_report
+        correlated_reports.append(current_report)
+    if correlated_reports:
+        title = 'TruSTAR correlated reports'
+        entry = {
+            'Type': entryTypes['note'],
+            'Contents': correlated_reports,
+            'ContentsFormat': formats['json'],
+            'ReadableContentsFormat': formats['markdown'],
+            'HumanReadable': tableToMarkdown(title, correlated_reports)
+        }
+        return entry
+    return 'No reports were found.'
+
+
+def search_reports(search_term, enclave_ids):
+    response = ts.search_reports(search_term, enclave_ids)
+    reports = []
+    report_context = []
+    for report in response:
+        current_report = report.to_dict(remove_nones=True)
+        current_report['updated'] = normalize_time(current_report['updated'])
+        current_report['created'] = normalize_time(current_report['created'])
+        current_report['timeBegan'] = normalize_time(current_report['timeBegan'])
+        reports.append(current_report)
+        report_context.append({
+            'reportTitle': current_report['title'],
+            'id': current_report['id']
+        })
+        if 'reportBody' in current_report:
+            report_context[0]['reportBody'] = current_report['reportBody']
```

Comment:
why are you appending to the first context and not the one relevant for the current report?